README.md

```diff
@@ -57,6 +57,8 @@ Installation
         $ python manage.py migrate
     ```
 
+5. Change the contact information shown in the app by modifying the
```

Comment:
Not sure if this is the best wording--to me this implies the person deploying should manually go in and change the contact_info.html file in place, but then if that person later wants to upgrade the Nomination app, s?he would gave to change the template again. So _if_ we want to go the not-using-settings route, it might be better to tell them to override the template instead.